WKWebView.ios.js

```diff
@@ -77,6 +82,12 @@ class WKWebView extends React.Component {
   static JSNavigationScheme = JSNavigationScheme;
   static NavigationType = NavigationType;
 
+  static setConfiguration = (config: Configuration) => {
+    if (WKWebViewManager.setConfiguration) {
+      WKWebViewManager.setConfiguration(config);
+    }
+  }
+
```

Comment:
Does this also need to be wired up to be configured from the `props` sent to the component?  

Such that we can do:

`<WKWebView allowsInlineMediaPlayback={true} />` 

for example?